addons/hr_holidays/report/hr_leave_report_calendar.py

```diff
@@ -55,6 +57,11 @@ def _read(self, fields):
                 self.env.cache.set(record, name_field, record.name.split(':')[-1].strip())
         return res
 
+    @api.depends('state')
+    def _compute_is_hatched(self):
+        for holiday in self:
+            holiday.is_hatched = holiday.state not in ['refuse', 'validate']
+
```

Comment:
As you are in a report, and the computation of `is_hatched` is easy, it's better to include it in SQL query in init.

Reply:
Done, thanks for the review